src/reporter.js

```diff
@@ -197,6 +204,26 @@ function CoverageIstanbulReporter(baseReporterDecorator, logger, config) {
       });
     });
 
+    if (
+      Object.keys(temporaryFileDateResult).length > 0 &&
+      coverageConfig.tempDirectory
+    ) {
+      const tempFolder = coverageConfig.tempDirectory;
+
+      if (fs.existsSync(tempFolder) === true) {
+        fs.removeSync(tempFolder);
+      }
+
+      fs.mkdirSync(tempFolder, { recursive: true });
+
+      fs.writeFileSync(
+        path.join(tempFolder, `${Date.now()}.json`),
+        JSON.stringify(temporaryFileDateResult)
+      );
+    } else {
+      log.warn('No coverage data found');
```

Comment:
This will always log when you're not using the `tempDirectory` option. Can you also make it `debug` level as well?